mem_storage/src/main/java/com/splicemachine/storage/MemFileSystem.java

```diff
@@ -257,8 +259,40 @@ public boolean exists(){
         }
 
         @Override
-        public FileInfo[] listRecursive(){
+        public FileInfo[] listFilesRecursive(){
             throw new UnsupportedOperationException("IMPLEMENT");
         }
+        @Override
+        public FileInfo[] listDir(){
+            try {
+                return Files.list(p).map( path -> new PathInfo(path) ).toArray(PathInfo[]::new);
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
I wouldn't do that, instead I might throw a `RuntimeException` and/or use our logging mechanism to log the exception. IIRC `printStackTrace` writes to `stderr` which could be not inline with our logging framework.

Reply:
done